File: 26-RemoveDuplicatesFromSortedArray/26-RemoveDuplicatesFromSortedArray.py
# Last updated: 2/25/2026, 9:10:51 AM
class Solution(object):
    def removeDuplicates(self, nums):
        duplicates = []
        c=0
        for i in range(0,len(nums)):
            if nums[i] not in duplicates:
                nums[c]=nums[i]
                c+=1  
                duplicates.append(nums[i])      
        return c

File: 26-RemoveDuplicatesFromSortedArray/26-RemoveDuplicatesFromSortedArray.java
// Last updated: 2/25/2026, 9:10:46 AM
class Solution {
    public int removeDuplicates(int[] nums) {


        int n=nums.length;
        int[] p=new int[n];
        int k=1;
        for(int i=1;i<n;i++){
            if(nums[i]!=nums[k-1]){
                nums[k]=nums[i];
                k++;
            }
        }

        

        return k;

    }
}